sources/Interop/Windows/um/dxcapi/Windows.Manual.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public static unsafe partial class Windows
+    {
+        [NativeTypeName("const UINT32")]
+        public static readonly uint DxcValidatorFlags_Default = 0;
```

Comment:
These and the others should be `const`? It's only the Guids that need to be `static readonly`